webpack-array-methods.config.js

```diff
@@ -6,6 +6,8 @@ const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const OptimizeCssAssetsWebpackPlugin = require('optimize-css-assets-webpack-plugin');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const TerserWebpackPlugin = require('terser-webpack-plugin');
 const path = require('path');
```

Comment:
"import/no-extraneous-dependencies": [
        "error", {
           "devDependencies": true
        }
    ]

Reply:
I found even a better way. Instead of disabling any warnings about imported dev dependencies, only disable them for the tests and webpack configs:

 "import/no-extraneous-dependencies": [
      "error",
      { "devDependencies": ["**/webpack*.config.js", "**/*.test.js", "**/*.spec.js"] }
    ]